NEWS.md

```diff
@@ -1,3 +1,39 @@
+# serocalculator 1.1.0
+
+* Deprecated `llik()` function and replaced by `log_likelihood()`
```

Comment:
Let's describe this as "Renamed `llik()` to `log_likelihood()` to create a more consistent API." 

Similarly for the other functions.

Reply:
This is done.